bioptim/limits/path_conditions.py

```diff
@@ -687,6 +700,18 @@ def concatenate(self, other: "InitialGuess"):
             interpolation=self.init.type,
         )
 
+    def scale(self, scaling: float) -> "InitialGuess":
+        """
+        Scaling an InitialGuess
+
+        Parameters
+        ----------
+        scaling: float
+            The scaling factor
+        """
+        self.init = self.init / scaling
```

Comment:
Same concern about raising a ValueError as Bounds

Reply:
By adding extra tests at the declaration of the scaling, I normally made sure that the dimensions are correct here